### PR TITLE
Upgrade to 2023.06

### PR DESCRIPTION
**Pull request by montjoie**

### CODE REVIEW

File: boards-ci-2023.06.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (aliceinwire) [inline boards-ci-2023.06.yaml:1]: why using a version number in the settings file? settings files shouldn't be versioned.

File: lava-master/entrypoint.d/01_setup.sh [hunk 2]
```diff
@@ -51,8 +51,58 @@ else
...
+		echo "============================="
+		sudo -u postgres psql lavaserver -c "SELECT * from django_migrations where app = 'lava_scheduler_app';"
+		echo "============================="
+		sudo -u postgres psql lavaserver -c "SELECT * from django_migrations where app = 'lava_results_app';"
+		echo "============================="
+	else
+		echo "DEBUG: no spetial handling for DB"
+	fi
+else
+	echo "DEBUG: no spetial handling for DB"
```
Comment (aliceinwire) [inline lava-master/entrypoint.d/01_setup.sh:99]: `spetial` -> special? spatial?

not sure what you are meaning by `no spetial handling for DB`

File: boards-ci-2023.06.yaml (hunk 1 above)
Comment (montjoie) [reply boards-ci-2023.06.yaml:1]: This file is used for CI.

File: lava-master/entrypoint.d/01_setup.sh (hunk 2 above)
Comment (montjoie) [reply lava-master/entrypoint.d/01_setup.sh:99]: i have rewrite this comment. Thanks

**Pull request merged**